### DISCUSSION TITLE
Can I underline the title?

### DISCUSSION THREAD

**Discussion by wbh3110** [nhn/tui.calendar#1309] [Q&A] [answered]
![notitle](https://user-images.githubusercontent.com/62129601/198256082-a4897fd8-8b90-4b58-9642-e95d9f10c2eb.png)

I'm developing with vue.js 2.x

is this possible?

**Comment by adhrinae** [comment, marked as answer]
Try out the `allday` template to customize the title.

https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/template.md#allday

**Reply by wbh3110** [reply]
Thanks for your kind explanation.
